docs/api/blockchain/accounts.mdx

```diff
@@ -956,6 +956,13 @@ _Query Parameters_
 
 200: OK
 
+:::note Transaction Units
+
+Remember that transaction units are in `bones`. Read more
+[here](/api/blockchain/introduction/#transaction-units)
+
+:::
+
```

Comment:
```suggestion
import TransactionUnitsSnippet from "snippets/transaction-units-snippet.mdx"

<TransactionUnitsSnippet />
```

Reply:
Just a suggestion but this should make it easier to maintain in case we want to change this in the future